lib/sportradar/api/nfl/season.rb

```diff
@@ -27,6 +29,9 @@ def set_weeks
         end
       end
 
+      # set_(conferences|divisions|teams) are all identical to the same methods in Hierarchy
+      # There is likely more overlap between Sportradar's NFL standings and NFL hierarchy APIs
```

Comment:
Yes, I'm purposefully holding on refactors like this until near the end. There are a few other big ones like the hash or array code that's copied and pasted throughout, those could easily get turned into a utility method that accepts args and does the same thing with a little metaprogramming